src/Common/src/System/Diagnostics/Debug.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Diagnostics
     /// <summary>
     /// Provides a set of properties and methods for debugging code.
     /// </summary>
-    static partial class Debug
+    public static partial class Debug
```

Comment:
Undo - some places (not yet on github) that use this common file do not want this public.